packages/toolpad-core/src/DashboardLayout/DashboardSidebarSubNavigation.tsx

```diff
@@ -236,17 +281,44 @@ function DashboardSidebarSubNavigation({
                 }}
               />
               {navigationItem.action && !isMini && isFullyExpanded ? navigationItem.action : null}
-              {navigationItem.children && !isMini && isFullyExpanded
-                ? nestedNavigationCollapseIcon
-                : null}
+              {navigationItem.children ? <ExpandMoreIcon sx={nestedNavigationCollapseSx} /> : null}
             </NavigationListItemButton>
+            {navigationItem.children && isMini ? (
+              <Grow in={navigationItemId === hoveredMiniSidebarItemId}>
+                <Box
+                  sx={{
+                    position: 'fixed',
+                    left: 62,
```

Comment:
Is there no avoidable way to hard code the left and `pl` values here? I could be wrong but this might create a future bug if we make the size of the mini sidebar configurable - perhaps we can link these values to that constant (if indeed they derive from it)